source/manual/get-started.html.md

```diff
@@ -229,6 +229,12 @@ If you don't need an optional dependency, you can pass the `-w` option:
 
 If these `bowl` commands fail, try the troubleshooting guide on [how to fix a broken bowl][bowl-error].
 
+Now visit this url once the app is running:
+
+`http://whitehall-admin.dev.gov.uk/`
+
+You should be able to see whitehall.
```

Comment:
I'd also suggest capitalising the "w" of "whitehall", as this is the name of the application.